Makefile

```diff
@@ -49,6 +49,7 @@ godeps:
 endif
 
 build: godeps
+	cat $(PWD)/patches/*.diff | patch -f -u -p1 -r- -d $(PWD)/../../../
```

Comment:
We can't modify what is supposed to be a clean source tree - this will mess with people's repos.

Reply:
Yea, everyone seems against overloading 'build'. I moved it to release-build, and added an 'undo' patches.